README.md

```diff
@@ -1,2 +1,86 @@
-# curriculum-transversal-skills
-This repository is created for learning git and git-hub at MicroverseInc
+<a id="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a id="about-project"></a>
+
+> Project created for learning Git & Github WorfFlow at Microverse
+
+Hello Microverse is an small document made with  educational purposes 
+
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+> HTML & CSS
+
+<!-- Features -->
+
+<!-- AUTHORS -->
+
+## 👥 Author <a id="authors"></a>
+
+> David Espino 
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/DavidEspinoG)
+- Twitter: [@twitterhandle](https://twitter.com/DavidEspinoG)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/davidespinog)
```

Comment:
@DavidEspinoG This section seems not fully customized.
![image](https://user-images.githubusercontent.com/61361037/211626921-1de87ec9-e2cd-48a3-b95b-57f8e3f1dada.png)
I think you can replace the `1` with your Name. And add GitHub, Twitter, and LinkedIn Handle to 2, 3 & 4 respectively.
For Example:
![image](https://user-images.githubusercontent.com/61361037/211627764-32e62d0f-13e2-4577-bc02-263e2630a20e.png)
But kindly don't forget to update the URLs of the social media handles too.